malmopy/README.md

```diff
@@ -12,72 +12,48 @@ training sessions.
 
 The CNTK Python binding can be installed by running the installation script 
 ([more information here](https://github.com/Microsoft/CNTK/wiki/Setup-CNTK-on-your-machine)).
-Follow the installation process after what you should be able to activate the cntk-pyXX environment
-(where XX is the python version).  
+Follow the installation process after what you should be able to import cntk.  
```

Comment:
Rephrase for clarity: "After following the installation process you should be able to import CNTK."